src/components/DifficultyExplanation/DifficultyExplanation.css

```diff
@@ -0,0 +1,19 @@
+.information{
+    display: flex;
+    position: absolute;
+    right: 25%;
+    width: 20%;
```

Comment:
since 20% will become really small in smaller windows, i think we should change this to ensure that even on smaller screens, the box will be at least a certain size. I suggest using the `max` function for this, maybe something like `max(20%, ___rem)`?